test/reporters/base.spec.js

```diff
@@ -423,6 +423,34 @@ describe('Base reporter', function () {
     expect(errOut, 'to be', '1) test title:');
   });
 
+  it('should append any error cause trail to stack traces', function () {
```

Comment:
[Testing] Let's add handling for the case of recursively referencing error stacks? As in A's cause is B and B's cause is A. It theoretically can happen from annoyingly crafted user code.

(this comment coming out of a discussion with all three of us)

Reply:
Done ✔️